rocky/tests/integration/test_reports.py

```diff
@@ -436,4 +430,3 @@ def test_multi_report(
         "Other": {"total": 2, "enabled": 2},
         "Web": {"total": 2, "enabled": 2},
     }
-    assert multi_data["recommendation_counts"] == {"Make sure there is a security.txt available.": 2}
```

Comment:
I think it would be good to stil test that KAT-NO-SECURITY-TXT works in the report.